### PR TITLE
Add krew-release-bot support

### PR DESCRIPTION
**Pull request by rajatjindal**
Add krew-release-bot support

### CODE REVIEW

**Comment by ahmetb** [comment]
I appreciate the work but can you only reduce this to krew release bot?

Sadly most github workflow steps you listed are buggy because of how bash works. Undefined variables, pipe errors, failed commands not terminating the shell etc. 

I don’t have time to iterate on those so let’s just limit this to bot release.

File: .krew/ctx.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        values:
+        - darwin
+        - linux
+    {{addURIAndSha "https://github.com/ahmetb/kubectx/archive/{{ .TagName }}.tar.gz" .TagName }}
+    bin: kubectx
+    files:
+    - from: kubectx-*/kubectx
+      to: .
+    - from: kubectx-*/LICENSE
+      to: .
```
Comment (ahmetb) [inline .krew/ctx.yaml]: Please instal .editorconfig to add newlines to files. 
Otherwise the bot will not be able to auto-approve this due to diff here.

**Comment by rajatjindal** [comment]
I've updated the PR as you suggested. thanks.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  push:
+    tags:
+    - 'v*.*.*'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Create Release
```
Comment (ahmetb) [inline .github/workflows/release.yml]: I actually think you don’t need to create a release. Every tag is sort of implicitly is a release. And we don’t upload files etc. yet the download link would still work without a release.

What do you think? Would that work with the bot?

Comment (rajatjindal) [reply .github/workflows/release.yml]: the bot at this moment expect release to be there. It also expected assets, which I removed while working on PR for kubectx. 

I think we can relax not having release too. I will need to make changes to action/bot for that. and I will do that this weekend.

Comment (ahmetb) [reply .github/workflows/release.yml]: I'm not really against creating a release, I guess it's ok to have this in this repo.

But I suspect many other bash-based repos will not bother creating releases. git tag && git push is good enough for many.

**Comment by ahmetb** [comment]
Hopefully next time we make a release I'll recall to see if this works. :)

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: release
+on:
+  push:
+    tags:
+    - 'v*.*.*'
```
Comment (ahmetb) [inline .github/workflows/release.yml]: please consider doing `*` here for other people's repos.

**Pull request merged**